packages/contracts/test/ProxyBorrowerWrappersScript.js

```diff
@@ -707,7 +707,7 @@ contract('BorrowerWrappers', async accounts => {
     const redeemedAmount = toBN(dec(100, 18))
     const sovFeeCollectorBalanceBefore = await wrbtcToken.balanceOf(sovFeeCollector)
     await th.redeemCollateral(whale, contracts, redeemedAmount)
-    const sovFeeCollectorBalanceAfter = await wrbtcToken.balanceOf(sovFeeCollector)
+    const sovFeeCollectorBalanceAfter = toBN(web3.utils.toBN(await web3.eth.getBalance(sovFeeCollector)));
```

Comment:
superfluous `toBN` conversions

Reply:
Updated in this [commit](https://github.com/DistributedCollective/zero/pull/174/commits/5f8563642a3cfaf87c3df95e7e3e2dcfa8f4eac3)